bot/exts/moderation/modlog.py

```diff
@@ -522,14 +523,57 @@ async def log_cached_deleted_message(self, message: discord.Message) -> None:
                 f"**Message ID:** `{message.id}`\n"
                 f"**Sent at:** {format_dt(message.created_at)}\n"
                 f"[Jump to message]({message.jump_url})\n"
-                "\n"
+                #"\n"
             )
 
+        # If the message is a reply, add the reference to the response
+        if message.reference is not None and message.reference.resolved is not None:
+            resolved_message = message.reference.resolved
+
+            if isinstance(resolved_message, discord.Message):
+                jump_url = resolved_message.jump_url
+                if resolved_message.channel.category:
+                    resolved_message_text = (
+                        f"{resolved_message.author} in "
+                        f"{resolved_message.channel.category}/#{resolved_message.channel.name}:"
+                    )
+                else:
+                    resolved_message_text = (
+                        f"{resolved_message.author} in "
+                        f"#{resolved_message.channel.name}:"
+                    )
+
+                #Shorten the message content if necessary
+                if len(resolved_message.clean_content) > 10:
+                    resolved_clean_content = resolved_message.clean_content[:10] + "..."
```

Comment:
I'm not sure whether to keep it or remove it. I think keeping it will make it bit easier for the mod to check who posted it without the need of clicking it. 

I'll add the shortener for the username!

Reply:
resolved this in [9d58aa1](https://github.com/python-discord/bot/pull/3106/commits/9d58aa15d25cb72513e26a220b02499a9b007725)